src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyCacheMaxSizeAction.java

```diff
@@ -133,53 +122,120 @@ public String toString() {
     return summary();
   }
 
-  public void setStepSizeForCache(final long stepSizeInBytes) {
```

Comment:
same as above

Reply:
The intent of adding final to these values to make them immutable. explicitly declare it as final so we won’t try to make any changes to that variable within the constructor. 

Based on our discussion. Removing final with setter's and retaining them in the constructors.